src/lib/accessibility/Behaviors/Chat/chatMessageBehavior.ts

```diff
@@ -5,15 +5,14 @@ import { FocusZoneTabbableElements, FocusZoneDirection } from '../../FocusZone'
 
 /**
  * @description
- * Adds role 'presentation' until we come up with final roles for chat.
  * Sets the message to be a focusable element.
  * Adds a vertical circular focus zone navigation where a user navigates using a Tab key.
  * Adds a key action which prevents up and down arrow keys from navigating in FocusZone, we only want a Tab key to navigate.
  */
 const chatMessageBehavior: Accessibility = (props: any) => ({
   attributes: {
     root: {
-      role: 'presentation',
+      role: undefined,
```

Comment:
shouldn't it be removed altogether then?

Reply:
that's a good question, I was also considering that. 
@jurokapsiar, do you think this ```role=presentation``` to be set by default in FocusZone? We always can add in behavior in cases where we need it

Reply:
After discussion with @jurokapsiar, we decided to remove that attribute from FocusZone. Thank you Roman, for raising this question, it always to have someone look at the situation with a fresh eye :)
So this attribute is added in the original FocusZone in fabric-ui, and they have some reasons for that, but we don't see the need for it right now, as in most cases we control ```role``` in behaviors.
Will update the PR